src/locales/en/common.json

```
@@ -521,7 +521,8 @@
     "emptyName": "Name cannot be empty",
     "invalid": {
       "address": "Invalid address",
-      "email": "Please enter a valid email"
+      "email": "Please enter a valid email",
+      "fiatValue": "Please, check amount. It may contain only numbers(0-9)."
```

Is is not correct as we also support decimal numbers like 1.50

```suggestion
      "fiatValue": "Please, check amount. It may contain only numbers(0-9)."
```

---

@IAmKio @mdjastrzebski I have taken this from design available on ticket. Kindly provide me the appropriate sentence.

---

@IAmKio pls help with the wording.

---

@divyaBacancy 
> Sorry, you can only enter numbers (0-9) here. Please try again.

---

Done